engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -239,12 +239,37 @@ object WorkflowDescriptor {
   val CallLogsDirOptionKey = "call_logs_dir"
   val OptionKeys: Set[String] = Set(WorkflowLogDirOptionKey, WorkflowOutputsOptionKey, CallLogsDirOptionKey)
 
+  // PBE:This is, frankly, an aberration. But don't worry - it's going away soon :D
+  // KILL IT WITH FIRE!!!
+  def getBackendFromOptions(optionsString: String): Backend = {
```

Comment:
I honestly think this should be defined elsewhere, (maybe in a separate file or in CromwellBackends, no opinions there, not sure if it can be for this PR though). It can then be extended in the subsequent PRs to initialize using reflection or returning a [potentially] list of backends that a WF may use (or anything else).